packages/discord.js/src/structures/GuildMember.js

```diff
@@ -492,6 +523,7 @@ class GuildMember extends Base {
     });
     json.avatarURL = this.avatarURL();
     json.displayAvatarURL = this.displayAvatarURL();
+    json.displayBannerURL = this.displayBannerURL();
```

Comment:
Any reason not to add `bannerURL` together with `displayBannerURL`, just like with the avatar?